packages/lesswrong/components/common/FundraisingThermometer.tsx

```diff
@@ -165,15 +177,15 @@ const FundraisingThermometer: React.FC<FundraisingThermometerProps & {classes: C
   const { LWTooltip } = Components
 
   return (
-    <div className={classes.root}>
+    <div className={classNames(classes.root, onPost && classes.onPost)}>
```

Comment:
I think @b0b3rt wants you to do this a different way

Like `{ [classes.onPost]: onPost }` or sthg?

Reply:
I forgot that we've moved on to this style (since it's more performant)

Reply:
K, I'll leave it as is